src/internal/src/mac/AudioDestinationMac.cpp

```diff
@@ -10,7 +10,25 @@
 
 #include <CoreAudio/AudioHardware.h>
 
-namespace lab {
+#include <rtaudio/RtAudio.h>
+
+namespace lab
+{
+
+static int
+NumDefaultOutputChannels() {
```

Comment:
This is a useful function and could be shared across mac/windows/linux

Reply:
It's specific to RtAudio. It would be good to have a generic RtAudio back end methinks, then this and some other logic would become common.